storage_controller/migrations/2025-02-11-144848_pageserver_use_https/up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE nodes ADD use_https BOOLEAN DEFAULT FALSE;
+ALTER TABLE nodes ALTER use_https SET NOT NULL;
```

Comment:
nit: any particular reason for not just adding `NOT NULL` in the `ADD` clause above? Or is this just autogenerated by Diesel or something?

It should be fine either way, mostly just curious.